src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16184,6 +16184,19 @@ instruct TailjmpInd(iRegPNoSpNoRfp jump_target, iRegP_R0 ex_oop)
   ins_pipe(pipe_class_call);
 %}
 
+// Forward exception.
+instruct ForwardExceptionjmp()
```

Comment:
```suggestion
instruct ForwardException()
```

Same for other AD files.

Reply:
We can't use the same name for Mach node as for Ideal node:
```
src/hotspot/cpu/x86/x86_64.ad(12596) Syntax Error: :duplicate name ForwardException for instruction
Error Context:  >>>(<<<)
src/hotspot/cpu/x86/x86_64.ad(12597) Syntax Error: :Identifier expected, but found '%{
  match(ForwardEx[...]'.
Error Context:  >>>%<<<{
```

Reply:
Ah, makes sense.